selfdrive/ui/onroad/driver_camera_view.py

```diff
@@ -9,13 +9,16 @@
 
 
 class DriverCameraView(CameraView):
-  def __init__(self, stream_type: VisionStreamType):
-    super().__init__("camerad", stream_type)
+  def __init__(self):
+    super().__init__("camerad", VisionStreamType.VISION_STREAM_DRIVER)
     self.driver_state_renderer = DriverStateRenderer()
 
   def render(self, rect):
     super().render(rect)
 
+    if rl.is_mouse_button_pressed(rl.MouseButton.MOUSE_BUTTON_LEFT):
+      return 1
+
```

Comment:
how does this work?

Reply:
Yes, return 1 to make it acts as a modal dialog and works with the modal overlay in gui_app. The overlay will be automatically dismissed when the function return>=0

Reply:
Checking mouse presses in render does feel weird. Maybe a callback instead? Or rename to update that calls render? Or you think this is best?

Reply:
How about renaming it to Dialog? All dialogs have a return value, and ModalOverlay relies on this return value to properly handle all popups. https://github.com/commaai/openpilot/pull/35478

Reply:
That would be better, yes